Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
 github "checkout/checkout-event-logger-ios-framework" "1.2.0"
-github "marmelroy/PhoneNumberKit" "3.3.7"
+github "marmelroy/PhoneNumberKit" "3.4.9"
```

Comment:
I've reviewed [changelog](https://github.com/marmelroy/PhoneNumberKit/releases) and I can see they mention some performance improvements and maintenance updates, so based on this seems fair to update.